src/dask_histogram/core.py

```diff
@@ -401,27 +401,28 @@ def optimize(
     keys: Hashable | list[Hashable] | set[Hashable],
     **kwargs: Any,
 ) -> Mapping:
-    if not isinstance(keys, (list, set)):
-        keys = [keys]
```

Comment:
The never happened/was uncovered?

Reply:
(or maybe `flatten` does the right thing already?)

Reply:
This is copying the function from dask-awkward, which is more thoroughly reviewed and battle hardened.

I know it's a bit cargo-cult, but it is also likely true in this case.